app/javascript/components/upload/ExpressionFileForm.js

```diff
@@ -105,15 +107,17 @@ function ExpressionFileInfoSelect({ label, propertyName, rawOptions, file, updat
   const selectOptions = rawOptions.map(opt => ({ label: opt, value: opt }))
   const selectedOption = selectOptions.find(opt => opt.value === file.expression_file_info[propertyName])
   return <div className="form-group">
-    <label>{label}</label><br/>
-    <Select options={selectOptions}
-      value={selectedOption}
-      placeholder="Select one..."
-      onChange={val => {
-        const expInfo = {}
-        expInfo[propertyName] = val.value
-        updateFile(file._id, { expression_file_info: expInfo })
-      }}/>
+    <label className="labeled-select">{label}
+      <Select options={selectOptions}
+        data-analytics-name={`expression-${propertyName}-select`}
```

Comment:
I (maybe naively) assume `propertyName` values are underscore-delimited as preferred by Ruby.  For consistency in Mixpanel, we should hyphen-delimit here.

```suggestion
        data-analytics-name={`expression-${_kebabCase(propertyName)}-select`}
```

(You'll need to import `_kebabCase`, of course.)